src/pseudopeople/interface.py

```diff
@@ -399,20 +387,74 @@ def generate_social_security(
     return _generate_dataset(DATASETS.ssa, source, seed, config, user_filters, verbose)
 
 
-def fetch_filepaths(dataset, source):
+def fetch_filepaths(dataset: Dataset, source: Path) -> Union[List, List[dict]]:
+    # returns a list of filepaths for all Datasets except 1040.
+    # 1040 returns a list of dicts where each dict is a shard containing a key for each tax dataset
```

Comment:
What do you mean by each dict is a "shard"?

Reply:
As in the shards for our full size run - one of the 334 shards. So the list will be [{tax_1040: 'file_1", tax_w2: "file_1", tax_dependents: "file_1"}, {tax_1040: "file_2", tax_w2: "file_2"....}]